fusionauth/helpers.go

```diff
@@ -16,6 +16,17 @@ func handleStringSlice(key string, data *schema.ResourceData) []string {
 	return handleStringSliceFromSet(data.Get(key).(*schema.Set))
 }
 
+func handleStringSliceFromList(list []interface{}) []string {
+	fmt.Println(list)
```

Comment:
Is this a debug statement that should be removed?

Reply:
Yes, indeed, thanks for catching this. I removed this :)